airflow/providers/amazon/CHANGELOG.rst

```diff
@@ -25,7 +25,52 @@
 
 Changelog
 ---------
-* ``A bug intoduced in provider-amazon version 8.0.0 caused all 'EcsRunTaskOperator' tasks to detach from the ECS task after 10 minutes and fail - even if the ECS task was still running. In this version we are fixing it by returning the default 'waiter_max_attempts' value to 'sys.maxsize'``
+
+8.7.0
+.....
+
+.. warning:: A bug introduced in version 8.0.0 caused all ``EcsRunTaskOperator`` tasks to detach from the ECS task
+  and fail after 10 minutes, even if the ECS task was still running.
+  In this version we are fixing it by returning the default ``waiter_max_attempts`` value to ``sys.maxsize``.
+
+Features
+~~~~~~~~
+
+* ``Add decorator for suppress optional internal methods in Amazon Provider (#34034)``
```

Comment:
I think this is misc, because this is only intend to use in internals of Amazon provider and not by end users

Reply:
Yes I wasnt sure about this one.
Shall I move it to misc or hide it from the change log?

Reply:
Oh, I forgot about hide option in change log so I guess better hide it.